src/index.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable array-callback-return */
+/* eslint-disable no-undef */
+/* All Functions */
+import './style.css';
+import editTasks from './js.modules/editFunction.js';
+import removeTasks from './js.modules/removeFunction.js';
+import updatingData from './js.modules/updatingData';
+import removeAllTasks from './js.modules/removeALlfunction';
+
+/* All HTML References */
+const taskInput = document.querySelector('input');
+const tasksContainer = document.querySelector('.tasks-container');
+const completedTasks = document.querySelector('button');
+
+/* Class Object */
+class MyList {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+const listArray = [];
+export default listArray;
+const addTask = (taskValue) => {
+  const newTaskContainer = document.createElement('div');
+  newTaskContainer.classList.add('new-task--container');
+  newTaskContainer.innerHTML += `
+    <input type="checkbox" class="check-task">
+    <span>${taskValue}</span>
+    <i class="fa-solid fa-ellipsis-vertical"></i>
+    <i class="fa-solid fa-trash-can"></i>
+  `;
```

Comment:
Please, try to avoid duplicating functions. In this case, you have the same in lines 60-66.